src/test/lib/all_parameter_variant_test.cpp

```diff
@@ -80,4 +83,59 @@ TEST_F(AllParameterVariantTest, GetCurrentValue) {
     EXPECT_EQ(value, 123.4);
   }
 }
+
+TEST_F(AllParameterVariantTest, ToString) {
+  {
+    const AllParameterVariant parameter(ParameterID{17});
+    EXPECT_EQ(to_string(parameter), "Placeholder #17");
+  }
+  {
+    const AllParameterVariant parameter(ColumnID{17});
+    EXPECT_EQ(to_string(parameter), "Col #17");
+  }
+  {
+    StorageManager::get().add_table("int_float", load_table("src/test/tables/int_float.tbl"));
+    const std::shared_ptr<StoredTableNode> int_float = StoredTableNode::make("int_float");
+    const LQPColumnReference col_a = {int_float, ColumnID{0}};
+    const LQPColumnReference col_b = {int_float, ColumnID{1}};
+
+    const AllParameterVariant parameter_col_a(col_a);
+    EXPECT_EQ(to_string(parameter_col_a), "a");
+
+    const AllParameterVariant parameter_col_b(col_b);
+    EXPECT_EQ(to_string(parameter_col_b), "b");
+  }
+  {
+    const AllParameterVariant parameter("string");
+    EXPECT_EQ(to_string(parameter), "string");
+  }
+}
+
+TEST_F(AllParameterVariantTest, ValuesNear) {
+  {
+    const AllParameterVariant parameter_0(ParameterID{17});
+    const AllParameterVariant parameter_1(ParameterID{16});
+
+    ASSERT_TRUE(all_parameter_variant_near(parameter_0, parameter_0, 17.0));
```

Comment:
Mmh, I don't think that this test makes too much sense. `all_parameter_variant_near` is unused anyway - just get rid of it.

Reply:
I have absolutely no idea. Close and wait until Moritz can concentrate on this?
He introduced the function with his work on the LQPs.

Reply:
These functions were introduced because when testing SQL -> LQP translation with floating point values in the SQL, you could never be sure which **precise** value the SQLParser would generate. I think I fixed this in the meantime by only using 0.5, 0.25, 0.125 etc in the tests - those have an unambiguous bit representation.

If they are unused, get rid of these functions

Reply:
Ok, so I remove both `all_type_variant_near` and `all_parameter_variant_near`, right?

Reply:
yepp